src/app/lecture/knowledge-node/notes/model/note.model.ts

```diff
@@ -0,0 +1,11 @@
+export class Note {
+  id: number;
```

Comment:
A note should be related to a knowledge node for which it is made. Add a KN ID and set it whenever a new note is created.

Reply:
I was thinking to forward the nodeId the same way we do in arrange task and question services.